common/source/docs/common-camera-shutter-with-servo.rst

```diff
@@ -22,67 +21,51 @@ This article explains what settings you need to configure for both servos and re
 Shutter configuration with Pixhawk or IOMCU Equipped Autopilots
 ===============================================================
 
-Pixhawk has 6 AUX Ports (AUX1-AUX6, referred to as RC9-RC14 in *Mission
+Pixhawk has 6 AUX Ports (AUX1-AUX6, referred to as SERVO9-SERVO14 in *Mission
 Planner*) that can be configured as :ref:`servos <common-servo>`,
 :ref:`relays <common-relay>`, or 
 :ref:`digital inputs or outputs <common-pixhawk-overview_pixhawk_digital_outputs_and_inputs_virtual_pins_50-55>`.
-The image and configuration below is for the Pixhawk with RC10/AUX2
-connected to camera control hardware and configured as either a servo or relay.
+The image and configuration below is for the Pixhawk with SERVO10 (labeled RC10/AUX2 on this autopilot) connected to camera control hardware and configured as either a servo or relay.
 
 .. figure:: ../../../images/Pixhawkdetailview.jpg
    :target: ../_images/Pixhawkdetailview.jpg
 
    Pixhawk Detail View highlighting AUXPorts
 
+.. note:: on autopilots not using an IOMCU (most that do, label outputs as MAIN/AUX), any output can be used for a relay or servo. See :ref:`common-gpios` for how to designate an output as a GPIO for relay use.
+
 .. tip::
 
-   From Plane 3.5.0 (at time of writing not in current Copter
-   3.3/Rover 3.5 releases) you can monitor and log *exactly* when the was
+   You can monitor and log *exactly* when the was
```

Comment:
little english issue here.  I guess it should be, "You can monitor and log *exactly* when the camera was triggered"